tasks/InsightAppSec/helpers/insightAppSecApi.ts

```diff
@@ -285,51 +285,56 @@ export default class InsightAppSecApi
         return new Promise(function (resolve, reject)
         {
             try
-            {
-                var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
-                let xhr = new XMLHttpRequest();
+            {   const axios = require('axios').default;
+
+                let axiosInst = axios.create({
+                    baseURL: endpoint,
```

Comment:
Is it possible to reuse this instance of axios instead of recreating every time? We configure the baseURL to be the common piece between requests (from the constructor), then when invoking the axiosInst below pass in in specific path (from the method params)?

Reply:
Reusable axios instance has been tested as working, but was unable to find any best practices relating to this